invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v7.0.0.json

```diff
@@ -0,0 +1,1491 @@
+{
+  "settings": {
+    "index.query.default_field": [
+      "id",
+      "metadata.title",
+      "metadata.title.original",
```

Comment:
the `title.original` is not part of the mapping, but looks like coming from the `dynamic` section, correct?

Reply:
It is part of the mapping
[See line 1164](https://github.com/inveniosoftware/invenio-rdm-records/pull/1774/files#diff-40a24708a1fed90c5895e1320701bb212c6230ad4bf6d9846a67e973cc5e9b23R1164)